CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
 Fix:  Fixed the bug about recovering ONTIMEINTERVAL subscriptions (Issue #693)
+Fix:  Fixed the crash in queryContext with a double geoscope (Issue #690)
```

Comment:
Well, the fix is valid for any invalid geoscope, not just this exact failure, right?
The 'exact' info is in the issue.
I changed the text to (simply):

Fixed the crash in queryContext with an invalid geoscope (Issue #690)

If you prefer your text, I have no problem in changing it